tests/contrib/langchain/test_langchain.py

```diff
@@ -407,7 +282,7 @@ def test_openai_chat_model_sync_call_39(langchain, request_vcr):
 @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Python 3.10+ specific test")
 @pytest.mark.snapshot(
     token="tests.contrib.langchain.test_langchain.test_openai_chat_model_generate",
-    ignores=["metrics.langchain.tokens.total_cost"],
+    ignores=["metrics.langchain.tokens.total_cost", "resource"],
```

Comment:
We do we need to ignore resource? Should we create multiple variants for this snapshot tests? Is this reasonable?

Reply:
The resource name is the class path, which is different for langchain 0.0 vs 0.1 😠